cfdm/data/data.py

```diff
@@ -1148,6 +1148,157 @@ def copy(self, array=True):
         '''
         return super().copy(array=array)
 
+    def creation_commands(self, name='data', namespace=None, indent=0,
+                          string=True):
+        '''Return the commands that would create the data object.
+
+    .. versionadded:: (cfdm) 1.8.7.0
+
+    :Parameters:
+
+        name: `str` or `None`, optional
+            Set the variable name of `Data` object that the commands
+            create.
+
+        {{namespace: `str`, optional}}
+
+        {{indent: `int`, optional}}
+
+        {{string: `bool`, optional}}
+
+    :Returns:
+
+        {{returns creation_commands}}
+
+    **Examples:**
+
+    >>> d = {{package}}.Data([[0.0, 45.0], [45.0, 90.0]],
+    ...                      units='degrees_east')
+    >>> print(d.creation_commands())
+    data = cfdm.Data([[0.0, 45.0], [45.0, 90.0]], units='degrees_east', dtype='f8')
+
+    >>> d = {{package}}.Data(['N/A', 'beta', 'gamma', 'delta'],
```

Comment:
Possibly I am missing something obvious, but is there a reason for the 'N/A' here? I;m guessing it is just to indicate that it will be masked so the value is irrelevant? Might be worth commenting as such for clarity as I was a little confused by it at first (could just be me, though).

Reply:
That's right. See https://github.com/NCAS-CMS/cfdm/pull/82/commits/951ba8bd89d5296db3283e195bae78e7a44148bc

Reply:
A simple solution, thanks.